test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -267,7 +271,7 @@ export default class BasePageObject {
   }
 
   formExists() {
-    return Selector('form').exists;
+    return Selector('form').visible;
```

Comment:
We should keep `.exists` because it checks for something different than `.visible` (sometimes we actually hide the form but it's still in DOM).

`.exists` doesn't follow selector timeouts like `.visible` - see https://github.com/DevExpress/testcafe/issues/5368#issuecomment-668448192

We can keep this the same but change the assertions (which follow assertion timeout settings and will retry up to the timeout). This would be like:
```js
t.expect(page.formExists()).ok();
```